docs/components/list-group.md

```diff
@@ -31,16 +31,16 @@ Add labels to any list group item to show unread counts, activity, etc.
 {% example html %}
 <ul class="list-group">
   <li class="list-group-item">
-    <span class="label label-default label-pill pull-xs-right">14</span>
     Cras justo odio
+    <span class="label label-default label-pill pull-right">14</span>
```

Comment:
(1) There are still `pull-left/right` *mixins*, but the `pull-left/right` *classes* are gone as of #18340. I just double-checked.
(2) I'm not 100% sure either. Will ask mdo when he reviews this.